packages/agent-toolkit/src/monday-graphql/queries.graphql.ts

```diff
@@ -54,6 +54,21 @@ export const getBoardSchema = gql`
   }
 `;
 
+
+export const getBoardAllActivity = gql`
```

Comment:
I think its time to have on the tool its own query, since this file gets too chaotic (so basically create a .graphql file in the tool folder maybe? 
wdyt?